README, man page, documentation, and wiki updates

kjy5 opened the pull request:
- Updated the [README](https://github.com/UWCubeSat/lost/blob/kjy5/setup/README.md), man pages ([pipeline](https://github.com/UWCubeSat/lost/blob/kjy5/setup/documentation/pipeline.man), [database](https://github.com/UWCubeSat/lost/blob/kjy5/setup/documentation/database.man)), and [HSL Wiki](https://wiki.huskysat.org/wiki/index.php/LOST#Building_and_Running_LOST) to have the correct information on running LOST with the new CLI
- Created a [Doxygen config](https://github.com/UWCubeSat/lost/blob/kjy5/setup/Doxyfile) which can be ran to create a Doxygen documentation website in `documentation/doxygen`
  - Created documentation for [`main.cpp`](https://github.com/UWCubeSat/lost/blob/kjy5/setup/src/main.cpp)
- Created issues board for people to attach issues to
- Discovered #17 
- Closes #13

kjy5 commented:
Oops, that must have been the auto-formatter. What formatting standard are we using?

markasoftware commented:
It's roughly the Google style guidelines but with tabs of width 4.

kjy5 commented:
Ah ok, I think I was using Google's except with tab width of 4. I will change this when I get the chance. How is everything else looking though?

markasoftware commented:
Also could you exclude the third-party CSS from github linguist so that the language breakdown stays accurate? https://stackoverflow.com/a/26948938 or https://github.com/github/linguist/blob/master/docs/overrides.md

markasoftware commented:
OH whoops I didn't realize that my comments were hidden until I submitted it as a review

kjy5 commented on Makefile:
```diff
@@ -66,6 +66,6 @@ $(TEST_BIN): $(TEST_OBJS)
 clean:
 	rm -f $(OBJS) $(DEPS) $(TEST_OBJS) $(MAN_HS)
-	rm -i $(BSC)
+	#rm -i $(BSC)
```
Can we remove the `-i` flag then since it causes the system to pause waiting for input before deleting that file

kjy5 commented on src/main.cpp:
```diff
@@ -16,29 +22,48 @@
+/**
+ * Build a database given arguments
+ * @param [in] values Parameters for building this database
+ * @sa
```
No, but I was mostly using GitHub Copilot to write them

kjy5 commented:
Updated based on comments.

Notes:
- There's probably a better way to do `make docs`. I didn't really understand how to utilize the existing documentation targets, but what I did does do the job. Would appreciate guidance on this.
- the µm character (initially represented as `\[m*]m`) isn't being recognized by groff, so I changed it to say "micrometers"

markasoftware commented:
I was not specific enough; there should be a Make target which generates the doxygen documentation. You don't need to touch the existing manpage processing.

kjy5 commented:
Making the doxygen would require the person have Doxygen installed... should I add that in as a dependency?

markasoftware commented:
The `docs` target will not run by default when you just run `make`, so it shouldn't make doxygen a dependency.

markasoftware commented:
You'd have to run `make docs` for it to run.

kjy5 commented:
Ok. That makes sense. I'm adding docker instructions to the README, then after that I think I've fixed all the changes. I think you have to approve the changes or something before the PR can be merged
